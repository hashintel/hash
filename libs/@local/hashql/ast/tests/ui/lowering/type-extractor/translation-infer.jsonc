//@ run: pass
//@ description: Tests translation of infer placeholders to core Infer type with hole IDs
["type", "Inferred", "_", "_"]
