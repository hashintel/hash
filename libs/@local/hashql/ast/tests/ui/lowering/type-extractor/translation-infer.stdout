Expr#4294967040@11
  ExprKind (Underscore)

------------------------

Inferred:0 = _0